### PR TITLE
Translated "lab041219". 

### PR DESCRIPTION
**Pull request by POP1912**
Translated the folder "lab041219" to English.

### CODE REVIEW

File: programming_lab/lab041219/albero_nari.py [hunk 1]
```diff
@@ -7,19 +7,19 @@
...
 
 
-def trova_massimo(albero):
-    max_rel = albero.valore
-    for figlio in albero.figli:
-        m = trova_massimo(figlio)
+def find_max(tree):
+    max_rel = tree.value
+    for sons in tree.sons:
+        m = find_max(sonso)
```
Comment (edoardottt) [inline programming_lab/lab041219/albero_nari.py:22]: typo here -> sons

File: programming_lab/lab041219/max_ric.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+max([3,4]) =        3 > max([4]) ?
+max([4]) =          4 !!!!
+"""
+
+
+def max_rico(l):
+    if len(l) == 1:
+        return l[0]
+
+    massimo_relativo = max_rico(l[1:])#!/usr/bin/env python3
```
Comment (edoardottt) [inline programming_lab/lab041219/max_ric.py:30]: I think these are two different files

File: programming_lab/lab041219/palindrome.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Dec  4 15:00:44 2019
+
+@author: angelo
+
+Write a paindrome recursive function that takes a string as input and returns True if it is a palindrome, False otherwise
```
Comment (edoardottt) [inline programming_lab/lab041219/palindrome.py:8]: typo --> palindrome

**Review by edoardottt — CHANGES_REQUESTED** [review]
Good job. Thank you

**Review by POP1912 — COMMENTED** [review]
Suggested changes have been made.

**Pull request merged**